R/add-only-insulin-purchase.R

```diff
@@ -0,0 +1,43 @@
+#' Add column for persons who only purchase insulin drugs.
```

Comment:
```suggestion
#' Add column describing if persons have purchased only insulin drugs or not
```
Or something like that. Otherwise, it sounds like the column is only added for persons who have only purchased insulin drugs :) 

I removed the period at the end bc they don't include that in the examples in the [R Packages](https://r-pkgs.org/man.html#sec-man-workflow) book.